Drivers/seds/AfecHwas/AfecHwas.xml

```diff
@@ -0,0 +1,562 @@
+            </Activity>
+            <Activity name="GetValue">
+              <Argument name="afec" type="AfecHwas" />
+              <Argument name="analogChannel" type="AfecHwasChannel" />
+              <Body>
+                <MathOperation shortDescription="Calculate analogChannel mask" outputVariableRef="channelMaskVar">
+                  <Operator operator="pow" />
+                  <Value value="2" />
+                  <VariableRef variableRef="analogChannel" />
+                </MathOperation>
+                <Assignment outputVariableRef="destinationAddressVar">
+                  <LongDescription>Assign destination register address</LongDescription>
+                  <VariableRef variableRef="afecVar.mInstanceCherAddress"></VariableRef>
+                </Assignment>
+                <SendCommandPrimitive shortDescription="Enable chosen analogChannel" interface="RawMemoryAccess" command="WriteWord">
+                  <ArgumentValue name="address">
+                    <VariableRef variableRef="destinationAddressVar" />
+                  </ArgumentValue>
+                  <ArgumentValue name="mask">
+                    <VariableRef variableRef="channelMaskVar" />
+                  </ArgumentValue>
+                  <ArgumentValue name="inputValue">
+                    <VariableRef variableRef="channelMaskVar" />
+                  </ArgumentValue>
+                </SendCommandPrimitive>
+                <MathOperation shortDescription="Negate analogChannel mask" outputVariableRef="bitsToSetVar">
+                  <Operator operator="subtract" />
+                  <Value value="4294967295" />
+                  <VariableRef variableRef="channelMaskVar" />
+                </MathOperation>
+                <Assignment outputVariableRef="destinationAddressVar">
+                  <LongDescription>Assign destination register address</LongDescription>
+                  <VariableRef variableRef="afecVar.mInstanceChdrAddress"></VariableRef>
+                </Assignment>
+                <SendCommandPrimitive shortDescription="Disable the rest of the channels" interface="RawMemoryAccess" command="WriteWord">
+                  <ArgumentValue name="address">
+                    <VariableRef variableRef="destinationAddressVar" />
+                  </ArgumentValue>
+                  <ArgumentValue name="mask">
+                    <Value value="4095" />
+                  </ArgumentValue>
+                  <ArgumentValue name="inputValue">
+                    <VariableRef variableRef="bitsToSetVar" />
+                  </ArgumentValue>
+                </SendCommandPrimitive>
+                <Assignment outputVariableRef="sourceAddressVar">
+                  <LongDescription>Assign destination register address</LongDescription>
+                  <VariableRef variableRef="afecVar.mInstanceChsrAddress"></VariableRef>
+                </Assignment>
+                <Iteration iteratorVariableRef="readBitsVar">

```

Comment:
I think this is an overinterpretation of SEDS spec. The proper way to do such wait should be via onTimer transition between discrete states (like waiting). I will add task to Jira to change this later in order not to stall. Please add a comment indicating a TODO.

Reply:
OK... I still don't see the todo, but maybe task on Jira will suffice... in fact, it should be better. Discussion was held, problem understood.